### PR TITLE
DATAGO-68138 | Added prisma vulnerability checker in gha pipeline

### PR DESCRIPTION
**Pull request by LewisKSaint**
### Changes

- Vulnerability fixes
- Automated vulnerability scans with GitHub Actions

Fixes [DATAGO-68138](https://sol-jira.atlassian.net/browse/DATAGO-68138)

[DATAGO-68138]: https://sol-jira.atlassian.net/browse/DATAGO-68138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/build-test-dev.yml [hunk 1]
```diff
@@ -78,10 +92,58 @@ jobs:
...
+          tags: |
+            gcr.io/${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}
+          push: true
+
+      - name: Run Vulnerability PreCheck for Prisma
+        uses: ./maas-build-actions/.github/actions/prisma-vulnerability-checker
+        with:
+          docker_image_to_check: gcr.io/${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}
+          prisma_repository_name: "${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}"
+          project_squad: "mission-control"
```
Comment (hossein-ashraf-kashani) [inline .github/workflows/build-test-dev.yml:141]: can this be changed to launch-pad?

File: .github/workflows/vulncheck_periodic.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+          tags: |
+            gcr.io/${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}
+          push: true
+
+      - name: Run Vulnerability PreCheck for Prisma
+        uses: ./maas-build-actions/.github/actions/prisma-vulnerability-checker
+        with:
+          docker_image_to_check: gcr.io/${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}
+          prisma_repository_name: "${{ env.GCLOUD_PROJECT_ID_DEV }}/${{ env.IMAGE_NAME }}"
+          project_squad: "mission-control"
```
Comment (hossein-ashraf-kashani) [inline .github/workflows/vulncheck_periodic.yml:81]: same here

File: config/manager/kustomization.yaml [hunk 3]
```diff
@@ -11,4 +11,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/solacedev/pubsubplus-eventbroker-operator
-  newTag: v1.0.0-rc.3
+  newTag: 1.0.2-dev
```
Comment (hossein-ashraf-kashani) [inline config/manager/kustomization.yaml:14]: what is this new tag?

Comment (LewisKSaint) [reply config/manager/kustomization.yaml:14]: basically, the docker image to be used, is autogenerated based on the image

**Review by hossein-ashraf-kashani — APPROVED** [review]

**Pull request merged**